### ISSUE TITLE
Accessibility issue no label, id not rendering on multiselect tag

### ISSUE THREAD

**Issue by wahiche-architech** [monterail/vue-multiselect#337]
So I have the multiselect working well, and so happy to have it.  One thing that has come up is that it fails accessibility checks based on it not having a label.  There is a way to make that work by adding the label and using bootstraps sr-only(screen reader only) class, but the label needs to be bound to the multiselect and this is done with the for="id of multiselect", problem is when I add an id attribute to the multiselect in the html it does not render the id.  Any idea how to do this?

**Comment by jag1989** [comment]
+1

**Comment by shentao** [comment]
My bad. Right now you can use the prop `:input-id` that will add the `id` to the input (so that it can work with a label).
This will be merged with `:id` prop for consisteny. Sorry about that!
